.travis.yml

```diff
@@ -8,19 +8,18 @@ cache:
 install:
   - pip3 install -r requirements.txt
 
-script:
-  - py.test -v --color=yes --cov=.
+jobs:
+  include:
+    - stage: test
+      script: py.test -v --color=yes --cov=.
+    - stage: deploy
+      skip_cleanup: true
+      provider: script
+      script: bash etc/deploy_docs.sh
 
```

Comment:
The ordering of stages need to be clearly specified. Please see [docs](https://docs.travis-ci.com/user/build-stages#Specifying-Stage-Order-and-Conditions).